examples/lifecycle-manager.yaml

```diff
@@ -75,12 +75,3 @@ spec:
             - serve
             - --queue-name=lifecycle-manager-queue
             - --region=us-west-2
-          volumeMounts:
-            - name: ssl-certs
-              mountPath: /etc/ssl/certs/ca-certificates.crt
-              readOnly: true
-          imagePullPolicy: "Always"
```

Comment:
We might want to leave imagePullPolicy there

Reply:
Yes added done. But I recommend to use docker image version tag instead of the latest version

Reply:
Since this is a generic example, latest tag is fine, you should use a version tag in your own implementation of installing the controller. It also seems we have now removed the region flag, we should leave it (even though people might have different region in mind) since it’s a mandatory flag

Reply:
sorry, i see region flag is still there, but we should have image tag as latest and bring back imagePullPolicy

Reply:
ok sure. Done!